fdbserver/BackupWorker.actor.cpp

```diff
@@ -742,11 +767,18 @@ ACTOR Future<Void> saveMutationsToFile(BackupData* self, Version popVersion, int
 		    .detail("File", logFiles[i]->getFileName());
 	}
 
+	// Fetch cipher keys if any of the messages are encrypted.
+	if (!cipherDetails.empty()) {
+		std::unordered_map<BlobCipherDetails, Reference<BlobCipherKey>> getCipherKeysResult =
```

Comment:
nit: `cipherKeys = wait(getEncryptCipherKeys(self->db, cipherDetails));` Do we need the local map variable?